From b70a3cb873613574e96329cf98ae3193b09c29bb Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:07:12 -0400
Subject: [usb] keyspan: remove dead debugging code

Message-id: <1410887463-20674-361-git-send-email-dzickus@redhat.com>
Patchwork-id: 95363
O-Subject: [RHEL7 PATCH 360/591] USB: keyspan: remove dead debugging code
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 36904592bc0d6da0e77278d9694e5e4c66bc0a11
Author: Johan Hovold <jhovold@gmail.com>
Date:   Wed Mar 12 19:09:43 2014 +0100

    USB: keyspan: remove dead debugging code

    Remove out-commented and ifdeffed debugging code.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <greg@kroah.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/keyspan.c b/drivers/usb/serial/keyspan.c
index 265c677..d3acaea 100644
--- a/drivers/usb/serial/keyspan.c
+++ b/drivers/usb/serial/keyspan.c
@@ -397,17 +397,6 @@ static void usa26_instat_callback(struct urb *urb)
 
  msg = (struct keyspan_usa26_portStatusMessage *)data;
 
-#if 0
- dev_dbg(&urb->dev->dev,
-  "%s - port status: port %d cts %d dcd %d dsr %d ri %d toff %d txoff %d rxen %d cr %d",
-  __func__, msg->port, msg->hskia_cts, msg->gpia_dcd, msg->dsr,
-  msg->ri, msg->_txOff, msg->_txXoff, msg->rxEnabled,
-  msg->controlResponse);
-#endif
-
- /* Now do something useful with the data */
-
-
  /* Check port number from message and retrieve private data */
  if (msg->port >= serial->num_ports) {
   dev_dbg(&urb->dev->dev, "%s - Unexpected port number %d\n", __func__, msg->port);
@@ -523,9 +512,6 @@ static void usa28_instat_callback(struct urb *urb)
   goto exit;
  }
 
- /*dev_dbg(&urb->dev->dev, "%s %12ph", __func__, data);*/
-
- /* Now do something useful with the data */
  msg = (struct keyspan_usa28_portStatusMessage *)data;
 
  /* Check port number from message and retrieve private data */
@@ -605,9 +591,6 @@ static void usa49_instat_callback(struct urb *urb)
   goto exit;
  }
 
- /*dev_dbg(&urb->dev->dev, "%s: %11ph", __func__, data);*/
-
- /* Now do something useful with the data */
  msg = (struct keyspan_usa49_portStatusMessage *)data;
 
  /* Check port number from message and retrieve private data */
@@ -1793,12 +1776,6 @@ static int keyspan_usa28_send_setup(struct usb_serial *serial,
  err = usb_submit_urb(this_urb, GFP_ATOMIC);
  if (err != 0)
   dev_dbg(&port->dev, "%s - usb_submit_urb(setup) failed\n", __func__);
-#if 0
- else {
-  dev_dbg(&port->dev, "%s - usb_submit_urb(setup) OK %d bytes\n", __func__,
-      this_urb->transfer_buffer_length);
- }
-#endif
 
  return 0;
 }
@@ -1976,13 +1953,6 @@ static int keyspan_usa49_send_setup(struct usb_serial *serial,
  err = usb_submit_urb(this_urb, GFP_ATOMIC);
  if (err != 0)
   dev_dbg(&port->dev, "%s - usb_submit_urb(setup) failed (%d)\n", __func__, err);
-#if 0
- else {
-  dev_dbg(&port->dev, "%s - usb_submit_urb(%d) OK %d bytes (end %d)\n", __func__,
-   outcont_urb, this_urb->transfer_buffer_length,
-   usb_pipeendpoint(this_urb->pipe));
- }
-#endif
 
  return 0;
 }
-- 
1.7.1